plugins/inputs/kubernetes/kubernetes.go

```diff
@@ -70,15 +78,83 @@ func (k *Kubernetes) Init() error {
 	}
 	k.labelFilter = labelFilter
 
+	if k.URL == "" {
+		k.InsecureSkipVerify = true
```

Comment:
Can this be configured by the end user? I am wondering if this should be an option.

Reply:
Kubelets are contacted by IP, their certificates are issued by https://kubernetes.io/docs/tasks/tls/certificate-rotation/ and  is therefore can't be verified with standard CA bundle